SimPEG/DCIP/DCIPUtils.py

```diff
@@ -0,0 +1,924 @@
+from SimPEG import np
+import BaseDC as DC
+import BaseDC as IP
+
+def getActiveindfromTopo(mesh, topo):
```

Comment:
nice! We could move this into mesh utils ? (this can be taken care of once it is on dev)